component/host.go

```diff
@@ -12,6 +12,7 @@ type Host interface {
 	//
 	// ReportFatalError should be called by the component anytime after Component.Start() ends and
 	// before Component.Shutdown() begins.
+	// Deprecated: [x.x.x] Use ReportComponentStatus instead (with an event component.StatusFatalError)
```

Comment:
Worth being more specific on where `ReportComponentStatus` can be found since it's not on the Host